scripts/gen.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 set -e
 
+# Clean directory and cache
+go clean -cache
+rm -rf $GOPATH/src/github.com/
```

Comment:
I appreciate this work to create a deterministic build environment, however this seems pretty disruptive if run locally. Is this intended for CI? Maybe a cleaner approach would be to include a go.mod file?

Reply:
I've removed this part of the script from the generator script and added it into a separate script.

I've also filed https://github.com/googleapis/google-cloudevents/issues/141, where we can add a mod file.

Hope this resolves the concern!